### PR TITLE
[doc] Adjust the data file section of the Quick Start chapter and rearrange the file classification

### PR DESCRIPTION
**Pull request by wanghui42**
version: 0.14
Excerpts from unpublished documents, and removed incorrect heading level of the document for #

### CODE REVIEW

File: docs/zh/UserGuide/QuickStart/Files.md [hunk 1]
```diff
@@ -19,27 +19,91 @@
...
+> 在 basedir/data/目录下
 
 数据文件存储了用户写入 IoTDB 系统的所有数据。包含 TsFile 文件和其他文件，可通过 [data_dirs 配置项](../Reference/DataNode-Config-Manual.md) 进行配置。
 
 为了更好的支持用户对于磁盘空间扩展等存储需求，IoTDB 为 TsFile 的存储配置增加了多文件目录的存储方式，用户可自主配置多个存储路径作为数据的持久化位置（详情见 [data_dirs 配置项](../Reference/DataNode-Config-Manual.md)），并可以指定或自定义目录选择策略（详情见 [multi_dir_strategy 配置项](../Reference/DataNode-Config-Manual.md)）。
 
-### 系统文件
+### TsFile
+> 在 basedir/data/sequence or unsequence/{StorageGroupName}/{TimePartitionId}/目录下
+1. {time}-{version}-{mergeCnt}.tsfile
```
Comment (qiaojialin) [inline docs/zh/UserGuide/QuickStart/Files.md:37]:
```suggestion
1. {time}-{version}-{inner_compaction_count}-{cross_compaction_count}.tsfile
```

File: docs/UserGuide/QuickStart/Files.md [hunk 2]
```diff
@@ -19,27 +19,98 @@
...
 
 Data files store all the data that the user wrote to IoTDB, which contains TsFile and other files. TsFile storage directory can be configured with the `data_dirs` configuration item (see [file layer](../Reference/DataNode-Config-Manual.md) for details). Other files can be configured through [data_dirs](../Reference/DataNode-Config-Manual.md) configuration item (see [Engine Layer](../Reference/DataNode-Config-Manual.md) for details).
 
 In order to support users' storage requirements such as disk space expansion better, IoTDB supports multiple file directories storage methods for TsFile storage configuration. Users can set multiple storage paths as data storage locations( see [data_dirs](../Reference/DataNode-Config-Manual.md) configuration item), and you can specify or customize the directory selection strategy (see [multi_dir_strategy](../Reference/DataNode-Config-Manual.md) configuration item for details).
 
-### System files
+### TsFile
+> under directory data/sequence or unsequence/{StorageGroupName}/{TimePartitionId}/
+
+1. {time}-{version}-{mergeCnt}.tsfile
```
Comment (qiaojialin) [inline docs/UserGuide/QuickStart/Files.md:38]:
```suggestion
1. {time}-{version}-{inner_compaction_count}-{cross_compaction_count}.tsfile
```

File: docs/zh/UserGuide/QuickStart/Files.md [hunk 3]
```diff
@@ -19,27 +19,91 @@
...
+    + 数据文件
+2. {TsFileName}.tsfile.mod
+    + 更新文件，主要记录删除操作
+
+### TsFileResource
+1. {TsFileName}.tsfile.resource
+    + TsFile 的概要与索引文件
+2. {TsFileName}.tsfile.resource.temp
+    + 临时文件，用于避免更新 tsfile.resource 时损坏 tsfile.resource
+3. {TsFileName}.tsfile.resource.closing
```
Comment (qiaojialin) [inline docs/zh/UserGuide/QuickStart/Files.md:47]: no more this file

File: docs/UserGuide/QuickStart/Files.md [hunk 4]
```diff
@@ -19,27 +19,98 @@
...
+    + modification file
+    + record delete operation
+
+### TsFileResource
+1. {TsFileName}.tsfile.resource
+    + descriptor and statistic file of a TsFile
+2. {TsFileName}.tsfile.resource.temp
+    + temp file
+    + avoid damaging the tsfile.resource when updating it
+3. {TsFileName}.tsfile.resource.closing
```
Comment (qiaojialin) [inline docs/UserGuide/QuickStart/Files.md:50]: no more this file

File: docs/zh/UserGuide/QuickStart/Files.md [hunk 5]
```diff
@@ -19,27 +19,87 @@
...
+## 数据文件
+> 在 basedir/data/目录下
 
 数据文件存储了用户写入 IoTDB 系统的所有数据。包含 TsFile 文件和其他文件，可通过 [data_dirs 配置项](../Reference/DataNode-Config-Manual.md) 进行配置。
 
 为了更好的支持用户对于磁盘空间扩展等存储需求，IoTDB 为 TsFile 的存储配置增加了多文件目录的存储方式，用户可自主配置多个存储路径作为数据的持久化位置（详情见 [data_dirs 配置项](../Reference/DataNode-Config-Manual.md)），并可以指定或自定义目录选择策略（详情见 [multi_dir_strategy 配置项](../Reference/DataNode-Config-Manual.md)）。
 
-### 系统文件
+### TsFile
+> 在 basedir/data/sequence or unsequence/{StorageGroupName}/{TimePartitionId}/目录下
```
Comment (qiaojialin) [inline docs/zh/UserGuide/QuickStart/Files.md:36]:
```suggestion
> 在 basedir/data/sequence or unsequence/{DatabaseName}/{DataRegionId}/{TimePartitionId}/目录下
```

File: docs/UserGuide/QuickStart/Files.md [hunk 6]
```diff
@@ -19,27 +19,93 @@
...
+1. Ration-{compressionRatioSum}-{calTimes}
+    + record compression ratio of each tsfile
+## Pre-write Log Files
 
 Pre-write log files store WAL files. It can be configured through the `wal_dir` configuration item (see [System Layer](../Reference/DataNode-Config-Manual.md) for details).
 
-### Example of Setting Data storage Directory
+> under directory basedir/wal
+
+1. {StorageGroupName}-{TsFileName}/wal1
```
Comment (qiaojialin) [inline docs/UserGuide/QuickStart/Files.md:106]:
```suggestion
1. {DatabaseName}-{TsFileName}/wal1
```

File: docs/UserGuide/QuickStart/Files.md [hunk 7]
```diff
@@ -19,27 +19,93 @@
...
+## Data Files
+> under directory basedir/data/
 
 Data files store all the data that the user wrote to IoTDB, which contains TsFile and other files. TsFile storage directory can be configured with the `data_dirs` configuration item (see [file layer](../Reference/DataNode-Config-Manual.md) for details). Other files can be configured through [data_dirs](../Reference/DataNode-Config-Manual.md) configuration item (see [Engine Layer](../Reference/DataNode-Config-Manual.md) for details).
 
 In order to support users' storage requirements such as disk space expansion better, IoTDB supports multiple file directories storage methods for TsFile storage configuration. Users can set multiple storage paths as data storage locations( see [data_dirs](../Reference/DataNode-Config-Manual.md) configuration item), and you can specify or customize the directory selection strategy (see [multi_dir_strategy](../Reference/DataNode-Config-Manual.md) configuration item for details).
 
-### System files
+### TsFile
+> under directory data/sequence or unsequence/{StorageGroupName}/{TimePartitionId}/
```
Comment (qiaojialin) [inline docs/UserGuide/QuickStart/Files.md:36]:
```suggestion
> under directory data/sequence or unsequence/{DatabaseName}/{DataRegionId}/{TimePartitionId}/
```

File: docs/UserGuide/QuickStart/Files.md [hunk 8]
```diff
@@ -19,27 +19,93 @@
...
+#### Upgrade
+> under directory basedir/system/upgrade
+
+1. upgrade.txt
+    + record which files have been upgraded
+
+#### Merge
+> under directory basedir/system/database/{Database}/
+
+1. merge.mods
```
Comment (THUMarkLau) [inline docs/UserGuide/QuickStart/Files.md:85]: For compaction, temporal file is ends with `.cross` or `.inner`, logs files is ends with `.inner-compaction.log` or `.cross-compaction.log`, mods file is ends with `.inner.mods` or `.cross.mods`.

File: docs/zh/UserGuide/QuickStart/Files.md [hunk 9]
```diff
@@ -19,27 +19,87 @@
...
+#### Upgrade
+> 在 basedir/system/upgrade 目录下
+1. upgrade.txt
+    + 记录升级进度
+
+#### Merge
+> 在 basedir/system/database/{Database}/目录下
+1. merge.mods
+    + 记录合并过程中发生的删除等操作
+2. merge.log
```
Comment (THUMarkLau) [inline docs/zh/UserGuide/QuickStart/Files.md:82]: Same as above.

File: docs/UserGuide/QuickStart/Files.md [hunk 10]
```diff
@@ -19,27 +19,93 @@
...
+    + modification file generated during a merge
+2. merge.log
```
Comment (THUMarkLau) [inline docs/UserGuide/QuickStart/Files.md:87]: There are also.meta files that are temporary files for metadata during the merge

**Review by THUMarkLau — CHANGES_REQUESTED** [review]

File: docs/UserGuide/QuickStart/Files.md (hunk 8 above)
Comment (wanghui42) [reply docs/UserGuide/QuickStart/Files.md]: I've been modified as I understand，could u review again

File: docs/UserGuide/QuickStart/Files.md [hunk 11]
```diff
@@ -19,27 +19,94 @@
...
+> under directory basedir/data/sequence or unsequence/{DatabaseName}/
+
+1. file suffixe with `.cross ` or `.inner`
+    + modification file generated during a merge
+2. file suffixe with `.inner-compaction.log` or `.cross-compaction.log`
+    + record the progress of a merge
+3. file suffixe with `.compaction.mods`
+    + temporary files, for storing temporary data
+4. file suffixe with `.meta`
+    + temporary files of metadata generated during a merge
```
Comment (THUMarkLau) [inline docs/UserGuide/QuickStart/Files.md:58]:
```suggestion
1. file suffixe with `.cross ` or `.inner`
    + temporary files of metadata generated in a compaction task
2. file suffixe with `.inner-compaction.log` or `.cross-compaction.log`
    + record the progress of a compaction task
3. file suffixe with `.compaction.mods`
    + modification file generated during a compaction task
4. file suffixe with `.meta`
    + temporary files of metadata generated during a merge
```

File: docs/zh/UserGuide/QuickStart/Files.md [hunk 12]
```diff
@@ -19,27 +19,89 @@
...
+### 与合并相关的数据文件
+> 在 basedir/data/sequence or unsequence/{DatabaseName}/目录下
+
+1. 后缀为`.cross ` 或者 `.inner`
+    + 记录合并过程中发生的删除等操作
+2. 后缀为`.inner-compaction.log` 或者 `.cross-compaction.log`
+    + 记录合并进展的日志文件
+3. 后缀为`.compaction.mods`
+    + 临时文件，用于存放临时数据
+4. 后缀为`.meta`的文件
```
Comment (THUMarkLau) [inline docs/zh/UserGuide/QuickStart/Files.md:55]: Save as above.

**Review by THUMarkLau — CHANGES_REQUESTED** [review]

File: docs/zh/UserGuide/QuickStart/Files.md [hunk 13]
```diff
@@ -19,27 +19,89 @@
...
+
+### 与合并相关的数据文件
+> 在 basedir/data/sequence or unsequence/{DatabaseName}/目录下
+
+1. 后缀为`.cross ` 或者 `.inner`
+    + 记录合并过程中发生的删除等操作
+2. 后缀为`.inner-compaction.log` 或者 `.cross-compaction.log`
+    + 记录合并进展的日志文件
+3. 后缀为`.compaction.mods`
+    + 临时文件，用于存放临时数据
```
Comment (THUMarkLau) [inline docs/zh/UserGuide/QuickStart/Files.md:54]:
```suggestion
    + 记录合并过程中发生的删除等操作
```

**Review by THUMarkLau — APPROVED** [review]

**Pull request merged**